packages/flutter/lib/src/cassowary/solver.dart

```diff
@@ -286,10 +397,33 @@ class Solver {
     return Result.success;
   }
 
+  /// Returns whether the given edit [Variable] is present in the solver.
   bool hasEditVariable(Variable variable) {
     return _edits.containsKey(variable);
   }
 
+  /// Suggest an updated value for the edit variable. The edit variable
+  /// must already be added to the solver.
+  ///
+  /// Suggestions update values of variables within the [Solver] but take into
+  /// account all the constraints already present in the [Solver]. Depending
+  /// on the constraints, the value of the [Variable] may not actually be the
+  /// value specified. The actual value can be read after the next
+  /// `flushUpdates` call. Since these updates are merely "suggestions", they
+  /// may not be at `Priority.required`.
```

Comment:
here and elsewhere, avoid "may not". This one seems like it should probably be "should not"?